accounts/templates/accounts/profile.html

```diff
@@ -0,0 +1,116 @@
+{% extends 'landing_page/base.html' %} {% load static %} {% block content %}
+
+<link rel="stylesheet" href="{% static 'accounts/css/profile.css' %}" />
+<link rel="stylesheet" href="{% static 'accounts/css/modal.css' %}" />
+
+<div id="message_div">
+  <h1>Hi {{request.user}}!</h1>
+</div>
+<div class="content_div">
+  <div id="user_info_div">
+    <div id="profile_pic_div">
+      <img
+        src="{% static 'landing_page/images/profile.png' %}"
+        class="profile_img"
+      />
+      <div id="info">
+        <h2>{{request.user}}</h2>
+        <p>Member since: {{request.user.date_joined}}</p>
```

Comment:
i think that models may have a 'created_date' field inherently, would need to look into it.
don't think we have a "date_joined" one?

Reply:
data_joined should be inherent to the user model specifically.